docs/releasing.md

```diff
@@ -0,0 +1,27 @@
+# Releasing
+
+
+To create a release manually there are two steps:
+
+
+1. version bumping
+2. create & publish distribution
+
+
+
+## Version bumping
```

Comment:
A link to docs/versioning.md here would be nice.

Reply:
agreed. I think i will move the instructions there and link to them